examples/fluids/qfunctions/stabilization.h

```diff
@@ -52,6 +53,25 @@ CEED_QFUNCTION_HELPER void Stabilization(NewtonianIdealGasContext gas, State s,
   }
 }
 
+CEED_QFUNCTION_HELPER void Stabilization(NewtonianIdealGasContext gas, State s,
+    CeedScalar Tau_d[3], State ds[3], CeedScalar U_dot[5],
+    const CeedScalar body_force[5], const CeedScalar x[3], CeedScalar stab[5][3]) {
+  // -- Stabilization method: none (Galerkin), SU, or SUPG
+  CeedScalar Y[5] = {0};
+  switch (gas->stabilization) {
+  case STAB_NONE:
+    break;
+  case STAB_SU:
+    FluxInviscidStrong(gas, s, ds, Y);
+    break;
+  case STAB_SUPG:
+    FluxInviscidStrong(gas, s, ds, Y);
+    for (CeedInt j=0; j<5; j++) Y[j] += U_dot[j] - body_force[j];
+    break;
+  }
+  StabilizationMatrix(gas, s, Tau_d, Y, x, stab);
```

Comment:
To avoid confusion, the `Y` here has units of `[conservative]/[time]` so it should have a different name.

Reply:
Oh, right. I meant to change that. `U_dot`?

Reply:
Nope, we have that already!

Reply:
`R` for residual is common in the literature.